plugins/org.jkiss.dbeaver.model.jdbc/src/org/jkiss/dbeaver/model/impl/jdbc/JDBCDataSource.java

```diff
@@ -475,10 +474,12 @@ public void shutdown(@NotNull DBRProgressMonitor monitor)
     }
 
     @Override
-    public void initialize(@NotNull DBRProgressMonitor monitor)
-        throws DBException
-    {
-        getDefaultInstance().initializeMetaContext(monitor);
+    public void initialize(@NotNull DBRProgressMonitor monitor) throws DBException {
```

Comment:
It changes default behavior. Are we sure?

Reply:
getAvailableInstances just returns getDefaultInstance copied to a list.
I suppose it doesn't change the behavior. If you're afraid it may broke smth, then I can add just null check, ok